src/features/dataset/DatasetHeader.tsx

```diff
@@ -62,14 +63,14 @@ const DatasetHeader: React.FC<DatasetHeaderProps> = ({
           )}
 
           <div className='text-2xl text-qrinavy-500 font-black group hover:text'>
-            {dataset.meta?.title || dataset.name}
+            {header.name}
           </div>
           {showInfo && (
             <div className='flex mt-3 text-sm'>
               <DatasetInfoItem icon='automationFilled' label='automated' iconClassName='text-qrigreen' />
               <DatasetInfoItem icon='disk' label='59 MB' />
```

Comment:
ahh, okay, so there are two pieces missing here, sorry I didn't catch this before:

1) we should only show the `DatasetInfoItem` that says "automated" if the version info has `workflowID` filled in
2) In the `DatasetInfoItem` with label '59 MB', that should be replaced by the actual size of the dataset taken from the version info, we also have a util function `fileSize` that you can import that displays the file size appropriately:
```js
<DatasetInfoItem icon='disk' label={`fileSize(header.bodySize)`} />
```